ts/modules/networkgraph/layouts.ts

```diff
@@ -206,6 +211,23 @@ extend(
 
             this.approximation = options.approximation;
         },
+        update: function (
+            this: Highcharts.NetworkgraphLayout,
+            options: Highcharts.NetworkgraphLayoutAlgorithmOptions,
+            redraw?: boolean
+        ): void {
+            this.options = merge(this.options, options);
+            if (pick(redraw, true) && this.chart) {
+                this.chart.redraw();
+            }
+        },
+        updateSimulation: function (
+            this: Highcharts.NetworkgraphLayout,
+            forced: boolean,
+            enable?: boolean
+        ): void {
```

Comment:
Currently, `updateSimulation(true)` is setting the simulation to false (`undefined` to be exact).

Yep, I had some doubts as well, currently it uses:
`this.enableSimulation = forced ? enable : this.options.enableSimulation;`

So what should happen in case when `updateSimulation(false, true)` is used? Maybe instead of `forced` it should be more like:
`this.enableSimulation = useDefaultVal ? this.options.enableSimulation : enable;`

Reply:
Ah I see, so we can say it's resetting flag to the default value. Yes, it makes sense.